modules/create-vm/cmd/create-vm/main.go

```diff
@@ -51,8 +50,8 @@ func main() {
 	if err := vmCreator.OwnVolumes(vm); err != nil {
 		exit.ExitFromError(OwnVolumesErrorExitCode, err)
 	}
-	runStrategy := cliOptions.GetRunStrategy()
```

Comment:
Why drop this check?

Reply:
We can specify the `RunStrategy` either through virtctl parameters or directly in a manifest. The `runStrategy`  checked in this condition is specified separately in the task spec. If we don't use the `RunStrategy` parameter of a task the string will be empty and it will depend only on `StartVM` flag. Also `StartVM` should not perform any action if `runStrategy` of a created VM is already set to Always. - https://kubevirt.io/user-guide/virtual_machines/run_strategies/#virtctl

Reply:
So this always tries to start the VM and mightt return an error from the API?

Reply:
It only tries to start the VM when `StartVM` param is specified. It should not return error, when the VM is already running nothing happens.